### ISSUE TITLE
Why isn't https://mithril.js.org/changelog.html up to v2.2.2?

### ISSUE THREAD

**Issue by ericop** [MithrilJS/mithril.js#2828]
Hi mithril team,
I have been away for a while (since 2.0.3) and saw we are on https://mithril.js.org/ . So I went to the [change log](https://mithril.js.org/changelog.html) to see what's new, but it didn't say. Can we get this updated?

**Comment by JAForbes** [comment]
When we switched to [pr-release](https://pr-release.org/) we switched from a handwritten change log to an automated one.  

The automated log is currently output here: https://github.com/MithrilJS/mithril.js/blob/release-v2/docs/recent-changes.md

This is also mirrored by the github release notes: https://github.com/MithrilJS/mithril.js/releases

The site now links directly to github releases when you click on the changelog in the nav -  but that [changelog.html ](https://mithril.js.org/changelog.html) link still resolves...  We should probably figure out a way to merge the past and present logs so it is consistent.

[pr-release](https://pr-release.org/) just prepends the release notes to the existing changelog file, so we could probably inject the old content at the bottom of that file.  PR release will leave the existing content untouched and simply prepend.  Then we can have the docs feed from the `recent-changes.md` file instead of the old `changelog.md` file.

**Comment by dead-claudia** [comment]
@JAForbes "We should probably figure out a way to merge the past and present logs so it is consistent." This right here is the hard part, how we *should* reconcile it.

Also, the migration of the changelog to the docs repo throws yet another wrench into this problem. This is why, for now, I'm just directing people to https://github.com/MithrilJS/mithril.js/releases.

**Comment by JAForbes** [comment]
Can we just `wget https://raw.githubusercontent.com/MithrilJS/mithril.js/refs/heads/release-v2/docs/recent-changes.md` in the docs CI?

**Comment by dead-claudia** [comment]
@JAForbes No, because the docs needs notified somehow of that new version.

I have Dependabot set to check daily and open an ungrouped Mithril pull request on update, but of course, ideally, there'd be some sort of automation there to directly push the version update.
